doc/source/whatsnew/v0.23.1.txt

```diff
@@ -45,7 +45,11 @@ Bug Fixes
 ~~~~~~~~~
 
 -
--
+
+Timedelta
+^^^^^^^^^
+- Bug in :class:`Timedelta`: where a numerical value with a unit would round values (:issue: `12690`)
```

Comment:
can you be a little bit precise here, e.g. i think a reader might not get what this change is